README.md

```diff
@@ -19,44 +19,45 @@ As of 06/11/2017, this project is in the "working prototype" phase of developmen
 
 1. [Raspberry Pi](https://www.raspberrypi.org/products/)
 1. Raspberry Pi [camera module](https://www.raspberrypi.org/products/camera-module-v2/)
-1. SD card with NOOBS (minimum 4GB)
-1. Network access to the Pi (wifi for Pi3, ethernet for Pi2)
+1. SD card with Raspbian GNU/Linux 10 OS (minimum OS and SD Card size 4GB)
+1. Network access to the Pi (WiFi for Pi3, WiFi/ethernet for Pi2)
```

Comment:
hmmm.....network access is technically required for setup I guess.

Reply:
remove WiFi for Pi2